src/ls/CodeActions.ml

```diff
@@ -0,0 +1,40 @@
+open Lsp
+open Types
+open Lsp_util
+
```

Comment:
A Toplevel comment in the file rougly explaining the goal of the module would be helpful.
See other files in semgrep as example (e.g., AST_generic.ml, but does not have have to be that long :) )